# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/crashtests.list
# Commit: ec05c5d6c331
# Full Hash: ec05c5d6c331b5602f55214a1f08ff0a76f6ad8e
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-03 09:42:08
# Description:
#   Bug 1765667 - Clamp surface size of content tiles for backdrop-filter r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D145047
# ==============================================================================

diff -r 40b512ed4dbf -r ec05c5d6c331 gfx/tests/crashtests/crashtests.list
--- a/gfx/tests/crashtests/crashtests.list	Tue May 03 01:38:57 2022 +0000
+++ b/gfx/tests/crashtests/crashtests.list	Tue May 03 01:56:24 2022 +0000
@@ -212,3 +212,4 @@
 load 1757002.html
 load 1758127-1.html
 load 1762973-1.html
+pref(layout.css.backdrop-filter.enabled,true) load 1765667.html